Amplify/Amplify.swift

```diff
@@ -33,37 +33,37 @@ public class Amplify {
     // ease of testing.
 
     /// - Tag: Amplify.Analytics
-    public static internal(set) var Analytics = AnalyticsCategory()
+    public internal(set) static var Analytics = AnalyticsCategory()
 
     /// - Tag: Amplify.API
-    public static internal(set) var API: APICategory = APICategory()
+    public internal(set) static var API: APICategory = .init()
```

Comment:
Just wondering why does this particular one was changed to `.init()` but not the others?

Reply:
it's because the type already explicitly defined, the constructor can be inferred.  I've updated it to match other properties for consistency.